src/mc_control/generic_gripper.cpp

```diff
@@ -3,15 +3,16 @@
  */
 
 #include <mc_control/generic_gripper.h>
+#include <mc_filter/utils/clamp.h>
+#include <mc_rtc/gui.h>
 #include <mc_rtc/logging.h>
 
 #include <cmath>
 #include <numeric>
 #include <tinyxml2.h>
 
 #ifndef M_PI
-#  include <boost/math/constants/constants.hpp>
-#  define M_PI boost::math::constants::pi<double>()
+#  define M_PI = boost::math::constants::pi<double>();
```

Comment:
This line shouldn't have a `;` at the end

Reply:
We should probably add it to `mc_rbdyn/constants.h`. This define has many occurences, and it'd make quite a lot of sense to have a single definition of `M_PI` across all of `mc_rtc`.
And probably rename `mc_rbdyn/constants.h` to `mc_rtc/constants.h` while we're at it since those are not necessarily specific to `RBDyn`.

Reply:
Done in 66e232b